src/status_im2/common/contact_list_item/view.cljs

```diff
@@ -25,28 +26,36 @@
         admin?                    (get admins current-pk)
         checked?                  (reagent/atom (if start-a-new-chat?
                                                   user-selected?
-                                                  member?))]
-    [rn/touchable-opacity
-     {:on-press #(rf/dispatch [:bottom-sheet/show-sheet
-                               {:content (fn [] [actions/actions item extra-data])}])
-      :style    {:position :absolute
-                 :right    20}}
-     (if (= icon :options)
-       [quo/icon :i/options {:size 20 :color (colors/theme-colors colors/neutral-50 colors/neutral-40)}]
-       [quo/checkbox
-        {:default-checked?    @checked?
-         :accessibility-label :contact-toggle-check
-         :disabled?           (and member? (not admin?))
-         :on-change           (fn [selected]
-                                (if start-a-new-chat?
-                                  (on-toggle true @checked? public-key)
-                                  (if-not member?
-                                    (if selected
-                                      (rf/dispatch [:select-participant public-key true])
-                                      (rf/dispatch [:deselect-participant public-key true]))
-                                    (if selected
-                                      (rf/dispatch [:undo-deselect-member public-key true])
-                                      (rf/dispatch [:deselect-member public-key true])))))}])]))
+                                                  member?))
+        on-check                  (fn [selected]
+                                    (if start-a-new-chat?
+                                      (on-toggle true @checked? public-key)
+                                      (if-not member?
+                                        (if selected
+                                          (rf/dispatch [:select-participant public-key true])
+                                          (rf/dispatch [:deselect-participant public-key true]))
+                                        (if selected
+                                          (rf/dispatch [:undo-deselect-member public-key true])
+                                          (rf/dispatch [:deselect-member public-key true])))))]
+    [:f>
+     (fn []
+       [rn/touchable-opacity
+        {:on-press #(rf/dispatch [:bottom-sheet/show-sheet
+                                  {:content (fn [] [actions/actions item extra-data])}])
+         :style    {:position :absolute
+                    :right    20}}
+        (if (= icon :options)
+          [quo/icon :i/options
+           {:size  20
+            :color (colors/theme-colors colors/neutral-50 colors/neutral-40)}]
+          (react/use-memo
```

Comment:
is it related to the issue? or its jut for performance ?

Reply:
are this sure it's a proper solution and not a hack ?

Reply:
still open question

Reply:
could we fix checkbox ?

Reply:
nice ,thanks

Reply:
what's the issue with checkbox?? and yes let's fix the problem at the source :)